yunismohamed/Questioner-API
# #163076711 Add integration to TravisCI and Coveralls.io

#### What does this PR do?
Add integration to TravisCI and Coveralls.io

#### Description of Task to be completed?
- [x] Integrate TravisCI
- [x] Integrate Coveralls.io
- [x] Add the requirements.txt file
#### How should this be manually tested?
To manually test this:

1. Clone the repository using:
`git clone https://github.com/yunismohamed/Questioner-API.git`
2. Through your terminal, navigate to the cloned repository on your local machine and open it.
3. While in the folder on your terminal, change branches to the `ch-integrate-travisci-163076711` using `git checkout ch-integrate-travisci-163076711`.
4. Confirm the contents of `.travis.yml`.

#### What are the relevant pivotal tracker stories?
[#163076711](https://www.pivotaltracker.com/story/show/163076711)
